apps/academy/src/pages/index.tsx

```diff
@@ -1,58 +1,63 @@
+import Image from "next/image";
 import Link from "next/link";
 import type { ReactElement } from "react";
 import { Icons, LearnWeb3Banner, PartnerBanner } from "ui";
 
+//  LearnWeb3Banner, PartnerBanner
 import PageSeoLayout from "@/components/PageSeoLayout";
 import type { NextPageWithLayout } from "@/pages/_app";
 
 const Home: NextPageWithLayout = () => {
   return (
-    <>
-      <section className="academy-grid app-container">
-        <div className="academy-grid-col-1">
-          <div className="hidden justify-center md:flex">
-            <div className="text-bttf-lg">
-              <h1>
-                Learn
-                <br />
-                web3__
-                <br />
-                with
-                <br />
-                friends
-              </h1>
+    <div className=" ">
+      {/* HeroSection */}
+      <div className=" mt-[150px] md:mb-[118px] md:mt-[230px]">
+        <div className="  flex flex-col items-center justify-center text-white">
+          <h1 className="text-bttf text-center text-5xl text-[50px] tracking-wide  lg:text-[100px]">
+            <span className=" md:ml-[50px]">LEArn</span>
+            <br />
+            <span>WEB3__</span>
+            <br />
+            <span className=" md:mr-[50px]">WITH</span>
+            <br />
+            <span className=" relative flex items-baseline justify-end gap-x-2 md:ml-[20px]">
+              FRIENDS
+              <span className=" -right-20 bottom-0 ml-2 h-[25px] w-[25px] overflow-hidden rounded-full md:absolute md:ml-5 md:h-[50px] md:w-[50px]">
+                <Image alt="devIcon" src={"/dd_nft2.png"} width={500} height={500} />
+              </span>
+            </span>
+          </h1>
+
+          <div className=" mb-[7px] mt-[87px] flex flex-col items-center justify-center md:mb-0 md:mt-[50px] ">
+            <div className="  max-w-[495px] px-10 text-center text-lg font-light   md:px-0 md:text-2xl">
+              <p>Become a web3 developer with Developer DAO.</p>
```

Comment:
In the figma the text is:

`Become a web3 developer with DeveloperDAO. `

![image](https://github.com/user-attachments/assets/26973dfa-715d-4333-809c-597ea76db9a9)

Can you update it as in the figma?

Reply:
Changed the text from `Developer DAO` to `DeveloperDAO`.